docs/OMS-Agent-for-Linux.md

```diff
@@ -132,19 +132,24 @@ Communication between the agent and OMS services can use an HTTP or HTTPS proxy
 
 **Proxy Configuration**
 The proxy configuration value has the following syntax:
-[protocol://][user:password@]proxyhost[:port]
+[protocol://][user:password@]proxyhoddst[:port]
 
 Property|Description
 -|-
 Protocol|http or https
-user|Optional username for proxy authentication
-password|Optional password for proxy authentication
+user|User for proxy authentication; can be placeholder text
```

Comment:
Correction; these changes were partially made. But you must rebase this branch against master to get the correct diff.